dockerfiles/dev-env/Dockerfile

```diff
@@ -30,7 +30,7 @@
 #
 
 # build from the base image
-FROM terraferma/base:fenics-2019.1.0
+FROM terraferma/base:fenics-2019.1.0-focal
```

Comment:
Think this can just be renamed to fenics-2019.1.0 and take over from the current version.

Reply:
You're call entirely.  I just  didn't want to suprise someone who had an old version and suddenly we swapped out a whole OS version (plus two releases of python3 on them)

Reply:
Good point.  Agreed, let's leave the `-focal` tag.